### ISSUE TITLE
issue with Add/Update Authorized User button - Service Request, step 1: add users

### ISSUE THREAD

**Issue by ying-pbrc** [sparc-request/sparc-request#123]
This button was moved to the shared/user_list partial from shared/user_details_right and the local variable @can_edit seems to be missing hence it is always shown as "Add Authorized User" even when I am editing an authorized user profile. 
  
Please confirm that my observation is correct.  

Thanks, 
Ying

**Comment by robertjamesmiller** [comment]
We're experiencing the same issue plus we're unable to select a Role, do you have that problem too?

**Comment by robertjamesmiller** [comment]
In _user_list.html.haml, I think the styles should be removed from: 
%div{:style => 'padding-left:395px; padding-bottom: 55px; margin-top: -157px;'} 
and replaced with a CSS class (or no CSS which looks good to me). Should I create a pull request with the fix?

**Comment by ying-pbrc** [comment]
Yes,  because there are whole bunch of local variables are missing and it is always trying to add new user to protocol.

Also the button looks out of place on my screen. I can submit a pull request for you to review if you want to.


Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by robertjamesmiller** [comment]
If you've already coded a fix then please create a pull request. otherwise, I'll fix it Wednesday morning because we're trying to get v1.4.0 onto our production servers this week.

**Comment by ying-pbrc** [comment]
I will submit a pull request tomorrow morning.



Sent via the Samsung Galaxy S® 6, an AT&T 4G LTE smartphone


-------- Original message --------

**Comment by robertjamesmiller** [comment]
great, thanks. if you weren't already planning to do so, please remove the "style" attribute from %div{:style => 'padding-left:395px; padding-bottom: 55px; margin-top: -157px;'}

**Comment by ying-pbrc** [comment]
copied the following message from pull request #124: 
To correct myself, I found the problem was not due to any missing local variable, rather when updating an existing user, the user_list was not being reloaded.

Is there any reason why the Add Authorized User button always sitting there? Doesn’t it flow better where it was? I don't want to spend too much time to reinvent the engine and/or break anything else. I only fixed the issue by simply moving the .add-user div back to the user_details_right and updating its css to dark-blue-button.
